functions.py

```diff
@@ -585,7 +588,40 @@
         meta['featureArea'] = str(my_area)
     return meta, overlaps
 
-
+async def get_at_location(lat, lon, loci_type="any", count=1000, offset=0):
+    """
+    :param lat:
+    :type lat: float 
+    :param lon:
+    :type lon: float 
+    :param count:
+    :type count: int
+    :param offset:
+    :type offset: int
+    :return:
+    """
+    conn = await asyncpg.connect('postgresql://postgres:password@{}/mydb'.format(GEOBASE_ENDPOINT))
```

Comment:
Is exposing user:pass a problem?

Reply:
yes this is bad I will try to resolve in a future commit, the db will be behind a firewall and defaults should be changed and parameterized in anycase